robottelo/cli/hammer.py

```diff
@@ -92,7 +92,7 @@ def parse_help(output):
         'options': [],
     }
     option_regex = re.compile(
-        r'^ (-(?P<shortname>\w), )?(--(?P<name>[\w-]+))?'
+        r'^ (-(?P<shortname>\w), )?(--(\[.*?\])?(?P<name>[\w-]+))?'
```

Comment:
Why this re has changed ?

Reply:
@jyejare This was to take valid sub command if it starting with --[] (square bracket) and resolve list ot of index error see #6519